src/lib/openapi/spec/create-strategy-variant-schema.ts

```diff
@@ -0,0 +1,55 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const createStrategyVariantSchema = {
+    $id: '#/components/schemas/createStrategyVariantSchema',
+    type: 'object',
+    additionalProperties: true,
+    description:
+        '[WIP] A strategy variant allows to attach any data to strategies instead of only returning `true`/`false`. Strategy variants take precedence over feature variants.',
+    required: ['name', 'weight', 'weightType', 'stickiness'],
+    properties: {
+        name: {
+            type: 'string',
+            description: 'The variants name. Is unique for this feature toggle',
+            example: 'blue_group',
+        },
+        weight: {
+            type: 'number',
+            description:
+                'The weight is the likelihood of any one user getting this variant. It is a number between 0 and 1000. See the section on [variant weights](https://docs.getunleash.io/reference/feature-toggle-variants#variant-weight) for more information',
```

Comment:
Relates to the previous comment. Discard or accept depending on that.

```suggestion
                'The weight is the likelihood of any one user getting this variant. It is an integer between 0 and 1000. See the section on [variant weights](https://docs.getunleash.io/reference/feature-toggle-variants#variant-weight) for more information',
```